tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/PortalServerBehavior.java

```diff
@@ -694,6 +763,47 @@ private void setupAgent()
         }
     }
 
+    private void setupAriesJmxBundles()
+    {
+        final IPath modulesPath = getPortalRuntime().getPortalBundle().getLiferayHome().append( "osgi/modules" );
```

Comment:
hey Greg, If I put them into osgi/static , it will not be auto-deployed, so what about osgi/portal ? I will be auto-deployed.